setup.cfg

```diff
@@ -83,18 +84,23 @@ doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
 addopts = --doctest-rst -m "not figure" --doctest-ignore-import-errors
 asdf_schema_tests_enabled = true
 asdf_schema_root = sunpy/io/special/asdf/schemas/
+mpl-results-path = figure_test_images
+mpl-use-full-test-name = True
 markers =
     remote_data: marks this test function as needing remote data.
     online: marks this test function as needing online connectivity.
     figure: marks this test function as using hash-based Matplotlib figure verification. This mark is not meant to be directly applied, but is instead automatically applied when a test function uses the @sunpy.tests.helpers.figure_test decorator.
     flaky
     array_compare
+    mpl_image_compare
 remote_data_strict = True
 # Pin junit behaviour; we might want to update this to xunit2 at some point
 junit_family=xunit1
 filterwarnings =
     error
     error::sunpy.util.exceptions.SunpyDeprecationWarning
+    # Do not fail on pytest config issues (i.e. missing plugins) but do show them
+    always::pytest.PytestConfigWarning
```

Comment:
Does this fix https://github.com/sunpy/sunpy/issues/4612?

Reply:
I think it fixes some of the issues. I don't *think* it would fix the asdf issue tho.